projects/aca-content/src/lib/aca-content.routes.ts

```diff
@@ -40,6 +40,11 @@ import { ViewProfileComponent } from './components/view-profile/view-profile.com
 import { ViewProfileRuleGuard } from './components/view-profile/view-profile.guard';
 import { LoginComponent } from './components/login/login.component';
 import { Route } from '@angular/router';
+import { AppViewerComponent } from './components/viewer/viewer.component';
+import { SharedLinkViewComponent } from './components/shared-link-view/shared-link-view.component';
+import { PreviewComponent } from './components/preview/preview.component';
+import { TrashcanComponent } from './components/trashcan/trashcan.component';
+import { ManageRulesSmartComponent } from '@alfresco/aca-folder-rules';
 
 export const CONTENT_ROUTES: ExtensionRoute[] = [
```

Comment:
it does not look like a good idea to move part of the app to the lib

Reply:
I would say that this is a part of the lib, not application. 

e.g. in case if we want to use `aca-content` as a `npm package`, then we would have to copy this routing manually to other repositories (every cha)